### PR TITLE
Unpack extras in partner params

### PR DESCRIPTION
**Pull request by jumaallan**
Story: https://app.shortcut.com/smileid/story/xxx

## Summary

A few sentences/bullet points about the changes

## Known Issues

Any shortcomings in your work. This may include corner cases not correctly handled or issues related
to but not within the scope of your PR. Design compromises should be discussed here if they were not
already discussed above.

## Test Instructions

Concise test instructions on how to verify that your feature works as intended. This should include
changes to the development environment (if applicable) and all commands needed to run your work.

## Screenshot

If applicable (e.g. UI changes), add screenshots to help explain your work.

### CODE REVIEW

File: Sources/SmileID/Classes/Networking/Models/PartnerParams.swift [hunk 1]
```diff
@@ -32,3 +32,38 @@ public struct PartnerParams: Codable {
         )
     }
 }
+
+extension PartnerParams {
```
Comment (vanshg) [inline Sources/SmileID/Classes/Networking/Models/PartnerParams.swift:36]: nit: why make this an extension and now just put it in the class since it's in the same file?

File: Sources/SmileID/Classes/Networking/Models/PartnerParams.swift [hunk 2]
```diff
@@ -32,3 +32,38 @@ public struct PartnerParams: Codable {
...
+    struct DynamicCodingKey: CodingKey {
+        var stringValue: String
+        var intValue: Int?
```
Comment (vanshg) [inline Sources/SmileID/Classes/Networking/Models/PartnerParams.swift:39]: Why do we need intValue?

**Review by vanshg — APPROVED** [review]
some questions but LGTM
